drakcore/drakcore/analysis.py

```diff
@@ -46,6 +46,12 @@ def get_log_index(self, log_type, output_file):
             f"{self.uid}/index/{log_type}",
             output_file.name)
 
+    def get_wireshark_key_file(self, output_file):
+        """
+        Download tls session keys in format that is accepted by wireshark.
+        """
+        return self.minio.fget_object(self.bucket, f"{self.uid}/wireshark_key_file.txt", output_file.name)
```

Comment:
Are we 100% sure that the file is there? What if during analysis no traffic occurs?

Reply:
I've decided to leave this as it is and handle it in app.py (except NoSuchKey). Good catch.